sdv/multi_table/base.py

```diff
@@ -694,7 +694,14 @@ def load(cls, filepath):
                 The loaded synthesizer.
         """
         with open(filepath, 'rb') as f:
-            synthesizer = cloudpickle.load(f)
+            try:
+                synthesizer = cloudpickle.load(f)
+            except RuntimeError:
+                raise SamplingError(
```

Comment:
Could we only switch to the new error message after confirming the error that surfaced has that specific message about the current machine being CPU-only?